src/platform/common/cancellation.ts

```diff
@@ -2,18 +2,24 @@
 // Licensed under the MIT License.
 'use strict';
 
-import { CancellationToken, CancellationTokenSource } from 'vscode';
-import { BaseError } from '../errors/types';
+import { CancellationError, CancellationToken, CancellationTokenSource } from 'vscode';
 import { createDeferred } from './utils/async';
-import * as localize from './utils/localize';
+import { Common } from './utils/localize';
 
-/**
- * Error type thrown when canceling.
- */
-export class CancellationError extends BaseError {
```

Comment:
Removed our CancellationError, we weren't testing for this everywhere, hence we had a few weird behaviours when it came to handling of cancellation errors (basically we weren't handling it correctly as we weren't account for VSC and our cancellation error)
Now there's only one.